build.gradle.kts

```diff
@@ -125,6 +125,8 @@ dependencies {
     implementation("org.openrewrite:rewrite-maven:$rewriteVersion")
     runtimeOnly("com.fasterxml.jackson.core:jackson-core:2.12.+")
     runtimeOnly("org.openrewrite:rewrite-java-17:$rewriteVersion")
+    runtimeOnly("io.cucumber:cucumber-junit-platform-engine:7.+")
+    runtimeOnly("org.junit.platform:junit-platform-suite-api:latest.release")
```

Comment:
In general dependencies necessary for recipe testing should belong to the `testRuntimeOnly` group.